nomenclature/testing.py

```diff
@@ -21,6 +24,18 @@ def assert_valid_yaml(path: Path):
             error = True
             logger.error(f"Error parsing file {e}")
 
+        with open(file, "r", encoding="utf-8") as all_lines:
+            # check if any special character is found in the file
+            for index, line in enumerate(all_lines.readlines()):
+                for col, char in enumerate(line):
+                    if char not in printable:
+                        special_characters += (
+                            f"{file.name}, line {index + 1}, col {col + 1}. "
```

Comment:
Print error message with pointers to invalid characters as list.
```suggestion
                            f"\n - {file.name}, line {index + 1}, col {col + 1}. "
```